DNDinventory/ViewModel/MainViewModel.cs

```diff
@@ -121,6 +123,20 @@ public string SetFolderOutputTxt
             }
         }
 
+        private string settingsFileLocationTxt;
+        public string SettingsFileLocationTxt {
+            get {
+                return settingsFileLocationTxt;
+            }
+            set {
+                if (settingsFileLocationTxt != value) {
+                    settingsFileLocationTxt = $"Current settings file ({value})";
+                    OnPropertyChange("SettingsFileLocationTxt");
+                    Properties.Settings.Default.SettingsFileLocation = value;
```

Comment:
Latest used, not default by design ;-)

Reply:
intentional design, let the users test it first.

Reply:
Default is latest opened file. Agreed and see reactions